release_docs/RELEASE.txt

```diff
@@ -1117,6 +1117,14 @@ Bug Fixes since HDF5-1.12.0 release
 
     Configuration
     -------------
+    - Corrected pkg-config compile script
+
+      It was discovered that the position of the "$@" location in the compile script
+      may fail on some platforms and configurations. This was moved before the
+      pkg-config sub command.
```

Comment:
Suggestion to make the second sentence clearer - '"$@" was moved to a position before the pkg-config config sub command  in the pkg-config compile script.'